terralab/config.py

```diff
@@ -54,6 +55,7 @@ def load_config(
         ),
         teaspoons_api_url=teaspoons_api_url,
         server_port=int(server_port),
+        local_storage_path=f'{Path.home()}/{config["LOCAL_STORAGE_PATH"]}',
```

Comment:
im wondering if this should be the full path to the file like the other config values